core/src/main/java/io/fabric8/maven/core/service/openshift/OpenshiftBuildService.java

```diff
@@ -118,6 +110,11 @@ public void build(ImageConfiguration imageConfig) throws Fabric8ServiceException
         } catch (Fabric8ServiceException e) {
             throw e;
         } catch (Exception ex) {
+            // Log additional details in case of any IOException
+            if (ex != null && ex.getCause() instanceof IOException) {
+                log.error("Build for %s failed: %s", buildName, ex.getCause().getMessage());
+                logBuildFailure(client, buildName);
```

Comment:
We may need to implement new or update this method to log a correct error message. Right now it's logging a particular error message, IMHO which may be wrong. It should query the BuildConfig and log the status of last build happen.